FRC1076-2016-Competition/src/org/usfirst/frc/team1076/robot/statemachine/UDPAutonomous.java

```diff
@@ -25,9 +24,7 @@ public boolean shouldChange() {
 	
 	public MotorOutput driveTrainSpeed() {
 		sensors.interpretData();
-		if (Math.abs(sensors.getHeading() - sensors.currentHeading()) > 1) {

```

Comment:
Yeah, since this is getting removed, you probably don't need to review it. It might be best to review some of the changes all together, rather than commit by commit. I'm not completely sure though.